src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -24,6 +24,33 @@ public partial class MauiDrawingView
 		add => weakEventManager.AddEventHandler(value);
 		remove => weakEventManager.RemoveEventHandler(value);
 	}
+	
+	/// <summary>
+	/// Event raised when drawing started 
+	/// </summary>
+	public event EventHandler<MauiDrawingStartedEventArgs> DrawingStarted
+	{
+		add => weakEventManager.AddEventHandler(value);
+		remove => weakEventManager.RemoveEventHandler(value);
+	}
+	
+	/// <summary>
+	/// Event raised when drawing cancelled 
+	/// </summary>
+	public event EventHandler<MauiDrawingStartedEventArgs> DrawingCancelled
+	{
+		add => weakEventManager.AddEventHandler(value);
+		remove => weakEventManager.RemoveEventHandler(value);
+	}
+	
+	/// <summary>
+	/// Event raised when drawing 
+	/// </summary>
+	public event EventHandler<MauiOnDrawingEventArgs> Drawing
```

Comment:
We'll need to rename this using a past-tense verb.

Perhaps `Drawn`?

```suggestion
	public event EventHandler<MauiOnDrawingEventArgs> Drawn
```

In C#, events are fired because something in the past happened. It's best practice to use a past-tense verb (Drawing -> Drawn) for event names.

Reply:
It may be confused with a similar name DrawingLineCompleted. This event occurs exactly at the moment of drawing (on moving in a native implementation)